composer.json

```diff
@@ -67,13 +67,14 @@
         "ext-zip": "*",
         "ext-gd": "*",
         "phpunit/phpunit": "^4.8.36 || ^7.0",
-        "squizlabs/php_codesniffer": "^2.9",
+        "squizlabs/php_codesniffer": "^3.5",
         "friendsofphp/php-cs-fixer": "^2.2",
         "phpmd/phpmd": "2.*",
-        "phploc/phploc": "2.* || 3.* || 4.*",
+        "phploc/phploc": "2.* || 3.* || 4.* || 5.*",
         "dompdf/dompdf":"0.8.*",
         "tecnickcom/tcpdf": "6.*",
-        "mpdf/mpdf": "5.7.4 || 6.* || 7.*",
+        "mpdf/mpdf": "7.* || 8.*",
+        "rector/rector": "^0.8",
```

Comment:
In think this could be useful to add other rules as well. See Rector as a phpcsfixer-like tool.

Thanks for the feedback, I'll work on it as soon as your PR has been merged.